varcode/vcf_output.py

```diff
@@ -248,9 +249,11 @@ def get_sample_names():
         headers += ['FORMAT'] + sample_names
 
     unique_variants_list = merge_duplicate_variants()
+    # can't have more than one reference in VCF so pick the first one and pray
```

Comment:
instead of praying, throw an error? when would we ever want to support combining VCFs created against different references?

Reply:
Changed it to check for a unique reference